test/integration/specs/kbaseFeatureSet_test.js

```diff
@@ -37,9 +37,11 @@ describe('Test kbaseFeatureSet', () => {
             await expect(description).toHaveText(cellCase.description);
 
             // Test the 3rd row of the table.
-            const tableBody = await cell.$('[role="grid"] > tbody');
+            const tableBody = await cell.$('table.dataTable > tbody');
```

Comment:
`role="grid"` and `role="row"` are no longer put in by DataTables, and no longer recommended by the accessibility crowd: https://datatables.net/forums/discussion/68343/usage-of-the-role-attribtes